src/modules/sensors/failure_detector_HITL/failure_detector_HITL.hpp

```diff
@@ -0,0 +1,225 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2012-2024 PX4 Development Team. All rights reserved.
```

Comment:
Can use TII in the copyright text also here

Reply:
Thank you!